src/modify.js

```diff
@@ -92,14 +92,38 @@ exports.modifyImportMap = function (env, newValues) {
   }
 };
 
-exports.modifyService = function (env, serviceName, url, remove) {
+exports.modifyService = function (
+  env,
+  serviceName,
+  url,
+  remove,
+  packageDirLevel = 1
+) {
   return modifyLock(env, (json) => {
     const map = getMapFromManifest(json);
     if (remove) {
       delete map[serviceName];
     } else {
       map[serviceName] = url;
+
+      if (
+        (getConfig().packagesViaTrailingSlashes ||
+          !getConfig().hasOwnProperty("packagesViaTrailingSlashes")) &&
+        (url.match(new RegExp("/", "g")) || []).length > 1
```

Comment:
What's the purpose of this? To not attempt trailing slash if there are no directories in the main url? Could this be simplified to the following?

```suggestion
        url.contains("/")
```

Reply:
We discussed this over slack and I see now why this is here. It is for detecting the presence of multiple slashes. I was wrong on this.